.mailmap

```diff
@@ -2,20 +2,22 @@ Aaron Black <black27@llnl.gov>                  Aaron C. Black <black27@llnl.gov
 Adam J. Kunen <kunen1@llnl.gov>                 Adam J. Kunen <kunen1@sierra1626.llnl.gov>
 Arlie Capps <capps2@llnl.gov>                   Arlie Capps <capps2@cab669.llnl.gov>
 Arlie Capps <capps2@llnl.gov>                   Arlie Capps <capps2@cab689.llnl.gov>
+Arlie Capps <capps2@llnl.gov>                   Arlie Capps <48997041+agcapps@users.noreply.github.com>
```

Comment:
I am not sure we want to the `github` generated email since it is not a real e-mail

Reply:
It maps from right to left.  The github-generated names show up when you type
```
> git shortlog -ens
```

Reply:
Right. Those would only be merge commits though. I typically exclude them from the shortlog by the following:
```
 git shortlog -ens --no-merges
```

I actually have the following alias setup:
```
git config --global alias.hall-of-fame '-ens --no-merges'
```

Reply:
Please educate me. What is this file for?

Reply:
This file maps the identities for people that use multiple SSH keys (e.g., work on different machines), bound to different e-mails,  to a single identity.  See https://blog.developer.atlassian.com/aliasing-authors-in-git/ for more information.

Since the Github generated emails only occur for `merge` commits, we don't  necessarily need to maintain them in the .mailmap file, since merge commits are typically excluded from the shortlog using the `--no-merges` switch.

Reply:
Thanks for the explanation. The info in the file seems odd to me since 'cab' is extinct, 'sierra' isn't accessible, etc.

Reply:
But at some point in our history there was a "real" commit (i.e., not a merge commit) from that e-mail address. The mailmap file corrects this.

Reply:
I think the cab emails are from commits on platforms where people did not set up their git config author and email.

Reply:
@gzagaris -- thanks for the pointer about the `--no-merges` option.

Is there any harm in leaving these gh-generated commits in the mailmap file?

Reply:
No there is no harm in leaving those. But, I don't think we should try to maintain the gh-generated ones going forward.

Reply:
Thanks @gzagaris -- ok, leaving as is for now.

Incidentally, the github email addresses without a number appear to be legitimate commits.  For example, my most recent commit on this PR was created by editing the ``README`` file though gh's web interface.